test/mix/tasks/clone_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule Mix.Tasks.CloneTest do
+  use ExUnit.Case
+  import ExUnit.CaptureLog
+
+  alias Bypass
```

Comment:
Eh?

Reply:
it seems it was automatically added by LSP, and I didn't found